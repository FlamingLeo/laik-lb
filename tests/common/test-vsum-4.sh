#!/bin/sh
${LAUNCHER-./launcher} -n 4 ../../examples/vsum 100 | LC_ALL='C' sort > test-vsum-4.out
cmp test-vsum-4.out "$(dirname -- "${0}")/test-vsum-4.expected"
