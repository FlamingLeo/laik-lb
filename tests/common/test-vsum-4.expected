Id 0: partitial sums 4950, 300, 1225, 0
Id 1: partitial sums 0, 925, 1190, 528
Id 2: partitial sums 0, 1550, 1240, 1683
Id 3: partitial sums 0, 2175, 1295, 2739
My world ID 0, in shrinked group: -1
My world ID 1, in shrinked group: 0
My world ID 2, in shrinked group: 1
My world ID 3, in shrinked group: 2
Total sums: 4950, 4950, 4950, 4950
